custodia/server/__init__.py

```diff
@@ -100,7 +101,14 @@ def _create_plugin(parser, section, menu):
 
 
 def parse_config(args):
-    parser = ConfigParser(interpolation=ExtendedInterpolation())
+    defaults = {
+        'fqdn': socket.getfqdn(),
```

Comment:
The issues is using functions that depends on resolving reverse address is bad as it can produce weird random results some tiems if DNS is acting up or other similar stuff).
Given this value ends up in a configuration file directive I want something predicatable and changeable by the user by setting proper system configuration, not something that may depend on random network failures.